spring-boot/src/main/java/springboot/Reservation.java

```diff
@@ -205,7 +205,7 @@ public String createReservation(String checkIn, String checkOut, int roomNumber,
 
 
     //returns either a failure message or "success"
-    public String createReservation(Reservation newReservation) {
+    public String createReservation() {
```

Comment:
Im a little confused with this change. We are making it so that we can call createReservation on a reservation...wouldn't the reservation already been made if we are able to call a function on it? It seems like this function should be static, then we could call it anywhere, pass it a reservation and it would add it to the csv (databade).

Reply:
ok ur right i didnt think of that-- let me implement those changes